certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PPM Express",
+    "version": "v1.0",
+    "description": "PPM Express is Online Project Portfolio Management Software that was designed to easily integrate and orchestrate your existing projects in Microsoft Planner, Azure DevOps, Jira Software and Microsoft Project Online.",

```

Comment:
@vsorokin-ppmexpress please make the description more concise, it should describe the main purpose and value offered by your connector. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).